pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -529,6 +542,107 @@ const UnattendedRow = ({
     );
 };
 
+const CloudInitOptionsRow = ({
```

Comment:
This copies a lot of code from UnattendedRow. Could you please generalize/parameterize the two into one component?